pkg/bash/bash.go

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bash
+
+import (
+	"context"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+const (
+	initialBufferSize    = 1 << 10
+	finishMessage        = "gotestmd/pkg/suites/shell/Bash.const.finish"
+	cmdPrintStatusCode   = `echo -e \\n$?`
+	cmdPrintStdoutFinish = `echo ` + finishMessage
+	cmdPrintStderrFinish = cmdPrintStdoutFinish + ` >&2`
+)
+
+// Runner is an exported interface for bash runner
+type Runner interface {
+	// Dir returns the directory where the runner instance is located
+	Dir() string
+	// Close closes the bash process and all resources used by it
+	Close()
+	// Run runs the cmd
+	Run(cmd string) (stdout, stderr string, exitCode int, err error)
+}
```

Comment:
Can we cut this?
I don't think that this interface will be used somewhere.

Reply:
Done